### PR TITLE
Eighth bug fixes

### PR DESCRIPTION
**Pull request by naitian**

### CODE REVIEW

**Comment by naitian** [comment]
@ovkulkarni please pass this review thanks.

File: intranet/apps/eighth/views/admin/attendance.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def no_signups_roster(request, block_id):
             counselor = user.counselor
             row.append(counselor.last_name if counselor else "")
             row.append("{}".format(user.tj_email))
-            row.append(user.emails[0] if user.emails and len(user.emails) > 0 else "")
+            row.append(user.emails.first() if user.emails and user.emails.count() > 0 else "")
```
Comment (ovkulkarni) [inline intranet/apps/eighth/views/admin/attendance.py]: the `if user.emails` is probably unnecessary

File: intranet/apps/eighth/views/admin/groups.py [hunk 2]
```diff
@@ -113,7 +113,7 @@ def edit_group_view(request, group_id):
             "first_name": user.first_name,
             "last_name": user.last_name,
             "student_id": user.student_id,
-            "email": user.tj_email if user.tj_email else emails[0] if emails else "",
+            "email": user.tj_email if user.tj_email else emails.first() if emails else "",
```
Comment (ovkulkarni) [inline intranet/apps/eighth/views/admin/groups.py]: we want `if emails.count() > 0` instead of just `if emails`

**Comment by naitian** [comment]
Ok I'm gonna merge.

**Pull request closed without merging**